doc/Command_Index_User.tex

```diff
@@ -790,16 +790,23 @@ \subsubsection{End of Line Behavior}
 \subsubsection{Hyphenation}
 
 \macroname{\textbackslash gresethyphen}{\{\#1\}}{gregoriotex-main.tex}
-Tell Gregorio\TeX\ to when to place a hyphen between
-syllables in polysyllabic words in a score.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
+Tells Gregorio\TeX\ how to place a hyphen between syllables in polysyllabic words in a score.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
```

Comment:
I agree that reworking `\gresethyphen`'s internals would be a separate issue.  I'm just saying that the name `\ifgre@forcehyphen` makes me think you're doing what that issue would address, rather than what you're actually addressing here.

Basically the only issue I have right now is in your choice of names, but if you want to stick with it as is, I can confirm that everything works as expected and is ready for merge.